Cargo.toml

```diff
@@ -21,3 +21,6 @@ serde_derive = "1.0"
 [features]
 default = []
 alloc = ["serde/alloc"]
+
+# The std feature is necessary when compiling serde with std
+std = []
```

Comment:
I wonder if `std` should always depend on `alloc`

Reply:
Hmm, possibly?
In my particular case I don't use alloc, but shouldn't affect anything.